docs/Permissioning/Permissioning apis.md

```diff
@@ -0,0 +1,427 @@
+
+* Input: organization id to which the account belongs, account id, role id (it can be either org admin role or network admin role)
+* Output: Status of the operation
+* Example:
+```javascript
+> quorumPermission.assignAdminRole("ABC", "0xf017976fdf1521de2e108e63b423380307f501f8", "NWADMIN", {from: eth.accounts[0]})
+{
+  msg: "Action completed successfully",
+  status: true
+}
+```
+
+### quorumPermission.approveAdminRole 
+This api can be executed by the network admin to approve the organization admin or network admin role assignment to an account. The role is approved once majority approval is received.
+
+* Input: organization id to which the account belongs, account id
+* Output: Status of the operation
+* Example:
+```javascript
+> quorumPermission.approveAdminRole("ABC", "0xf017976fdf1521de2e108e63b423380307f501f8",  {from: eth.accounts[0]})
+
+{
+  msg: "Action completed successfully",
+  status: true
+}
+```
+
+### quorumPermission.addNode
+This api can be executed by the organization admin account to add a node to the organization or sub organization.
+
+* Input:  organization id or sub organization id, enode id
+* Output: Status of the operation
+* Example:
+```javascript
+> quorumPermission.addNode("ABC.SUB1.SUB2.SUB3", "enode://239c1f044a2b03b6c4713109af036b775c5418fe4ca63b04b1ce00124af00ddab7cc088fc46020cdc783b6207efe624551be4c06a994993d8d70f684688fb7cf@127.0.0.1:21006?discport=0&raftport=50407", {from: eth.accounts[1]})
+{
+  msg: "Action completed successfully",
+  status: true
+}
+```
+A node cannot be part of multiple organizations. 
+
+### quorumPermission.updateNodeStatus
+This api can be executed by the organization admin account to update the status of a node.
+
+* Input:  organization id or sub organization id, enode id, action (1 for deactivating the node, 2 for activating a deactivated node and 3 for blacklisting a node)
+* Output: Status of the operation
+* Example:
+```javascript
+> quorumPermission.updateNodeStatus("ABC.SUB1.SUB2.SUB3", "enode://239c1f044a2b03b6c4713109af036b775c5418fe4ca63b04b1ce00124af00ddab7cc088fc46020cdc783b6207efe624551be4c06a994993d8d70f684688fb7cf@127.0.0.1:21006?discport=0&raftport=50407",3, {from: eth.accounts[1]})
+{
+  msg: "Action completed successfully",
+  status: true
+}
+```
+Once a node is blacklisted no further action is possible on the same.
+
+### Roles
+
+#### Organization status types
+The table below indicates the numeric value for various organization statuses.
+
+| OrgStatus                 |           Value |
+| :-----------------------: | :-------------: |
+| NotInList                 |               0 |
+| Proposed                  |               1 |
+| Approved                  |               2 |
+| PendingSuspension         |               3 |
+| Suspended                 |               4 |
+| AwaitingSuspensionRevoke  |               5 |
```

Comment:
this list doesn't match with values defined in `core/types/permissions_cache.go`

Reply:
the only missing one is notinlist which is the default value. In permissions_cache.go we are starting from 1.